mqtt/mqtt-broker/src/persist.rs

```diff
@@ -313,14 +313,20 @@ where
                         path.clone(),
                     ))))
                 });
-                let file = OpenOptions::new()
+                let mut file = OpenOptions::new()
                     .read(true)
                     .open(&path)
                     .context(ErrorKind::Persist(ErrorReason::FileOpen(path)))?;
 
                 fail_point!("filepersistor.load.format", |_| {
                     Err(Error::from(ErrorKind::Persist(ErrorReason::Serialize)))
                 });
+
+                let mut version = [0; 100];
```

Comment:
`0u8`? and just curious why 100 byte? :)

Reply:
the read trait requires a u8, so the type will be inferred. 100 bytes was completely arbitrary, it seemed like a good number 🤷‍♂